tests/functional/z_cluster/nodes/test_nodes_restart_hci.py

```diff
@@ -376,6 +398,10 @@ def test_stop_provider_mgr_node_and_check_client_connection(
         logger.info("Check again that the client connection is OK")
         switch_to_client_for_function(wait_for_storage_client_connected)(timeout=120)
 
+        if onboard_new_client:
+            config.DEPLOYMENT["check_cnv_deployed"] = True
+            create_hypershift_clusters()
```

Comment:
please follow the example of how to use this factory fixture -> https://github.com/red-hat-storage/ocs-ci/blob/533ce37df7d731d6f02e242acfe8cba877eda2e4/tests/libtest/test_provider_create_hosted_cluster.py#L160

`cluster_names, ocp_version, odf_version, setup_storage_client, nodepool_replicas` are mandatory arguments

Reply:
Yes, but can't we use the default values here?

Reply:
Fixed.